pages/dkp/kaptain/2.0.0/migration/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+navigationTitle: Migration
+title: Migrate Kaptain
+menuWeight: 7
+excerpt: Migrate to Kaptain 2.0.0 on your DKP cluster
+beta: false
+enterprise: false
+---
+
+Due to a major update in Kaptain's lifecycle management tool, migrating your 1.x installation to 2.x is **not possible**. To ensure that you are up-to-date with the latest features, and can benefit from the most recent performance improvements, we recommend that you uninstall your Kaptain version and do a fresh install of the latest available version. 
+
+<p class="message--warning"><strong>WARNING: </strong>For this release, you will have to uninstall and install Kaptain to get to the latest version. Because this is a fresh install, ensure you back up all of your data before proceeding.</p>
+
+## Background
+
+Previous versions of Kaptain were installed with KUDO. From 2.0.0 and later, you can install Kaptain with Helm. As a result, migrating currently requires that you fully remove your prior Kaptain 1.x installation (including deployed models, profiles, notebooks, pipelines, etc.), and install Kaptain 2.0.0. Model artifacts are stored in your object store, therefore, you will not lose these as long as your model store is not deployed to the Kaptain namespace (that is, you are not using the MinIO server). However, **create a backup** of all your required data.
```

Comment:
We need instructions on how to both create AND restore a backup....

Reply:
@chessahoekstra @alembiewski @cneth 
I think this pplies here too: there is a way to back up and restore data. However, product and the team have decided not to document this as there is no way to guarantee it will work (at least for this release). Should we explicitly call this out? or leave a general statement as above? WDYT?